src/app/static/src/app/featureSwitches.ts

```diff
@@ -1,6 +1,6 @@
 const urlParams = new URLSearchParams(window.location.search);
-const shareProject = !!urlParams.get('shareProject');
-const promoteProject = !!urlParams.get('promoteProject');
+const shareProject = !urlParams.get('shareProject');
```

Comment:
share project isn't wired up (there's an outstanding PR) so lets leave this switch turned off